apps/labs/posts/scipy-array-api.md

```diff
@@ -0,0 +1,441 @@
+hero:
+  imageSrc: /posts/scipy-array-api/scipy-array-api-hero.png
+  imageAlt: 'An illustration of three cartoon characters - CuPy Cube, SciPy Snake and PyTorch Flame - together (happily!) under a tree with green leaves bearing the Consortium for Python Data API Standards logo ("DataAPIs") on a brown background. CuPy Cube and PyTorch Flame are made up of the CuPy and PyTorch logos respectively, with smiley faces and stick legs. SciPy Snake is a green snake with a red tongue, wearing sunglasses with SciPy logos for lenses. Some graphical elements on a white background surround the illustration.'
+---
+
+In this blog post I discuss how SciPy is harnessing the power of array-agnostic code to become
+more interoperable, and why that matters.
+
+---
+
+Hi 👋 I'm [Lucas Colley](https://github.com/lucascolley), a Computer Science & Philosophy student
+from the UK.
+This summer, I have had the amazing opportunity of working under the mentorship of
+[Ralf Gommers](https://github.com/rgommers), [Pamphile Roy](https://github.com/tupui) and
+[Irwin Zaid](https://github.com/izaid) in my Open Source internship at Quansight Labs.
+My project was to make progress towards SciPy's goal of adopting support for the array API standard,
+which will allow users to use SciPy functions with CuPy arrays, PyTorch Tensors and potentially
+many other array/tensor libraries in the future.
+
+This post stands on the shoulders of giants, including
+[Anirudh Dagar](https://anirudhdagar.github.io/)'s blog post
+["Array Libraries Interoperability"](https://labs.quansight.org/blog/array-libraries-interoperability)
+(from which my illustration takes much inspiration!) and [Aaron Meurer](https://www.asmeurer.com/blog/about/)'s
+SciPy 2023 talk ["Toward Array Interoperability in the Scientific Python Ecosystem"](https://youtu.be/16rB-fosAWw);
+for details of wider interoperability efforts and the array API standard,
+I urge you to check out those links!
+
+Here, I approach the topic from the focused perspective of a SciPy contributor.
+I hope that this post will be helpful for anyone who wants to contribute to SciPy's efforts to support
+the array API standard.
+
+---
+
+## SciPy and the Scientific Python Ecosystem
+
+SciPy is one part of a wide collection of Open Source tools, written in Python,
+which are fundamental to a range of work which relies on manipulating numbers with computers.
+This collection is known as the Scientific Python Ecosystem - a term which broadly
+encompasses Open Source Python projects which are community driven and used for science
+(see [scientific-python.org](https://scientific-python.org/about/) for more information).
+
+NumPy is the n-dimensional array library which SciPy is built to work with - it is SciPy's only non-optional
```

Comment:
Maybe worth referencing the NumPy paper here: https://www.nature.com/articles/s41586-020-2649-2 ?

Reply:
👍 also the SciPy one then